tests/functional/aws-node-sdk/test/multipleBackend/put.js

```diff
@@ -113,6 +128,22 @@ describe('MultipleBackend put object', () => {
                     });
                 });
             });
+
+            it('should put an object to AWS', done => {
+                const params = { Bucket: bucket, Key: key,
+                    Body: body,
+                    Metadata: { 'scal-location-constraint': 'aws-test' } };
+                s3.putObject(params, err => {
+                    assert.equal(err, null, 'Expected success, ' +
+                        `got error ${err}`);
+                    s3.getObject({ Bucket: bucket, Key: key }, (err, res) => {
+                        assert.strictEqual(err, null, 'Expected success, ' +
+                        `got error ${err}`);
+                        assert.strictEqual(res.ETag, `"${correctMD5}"`);
+                        done();
```

Comment:
You should also check the body in `res` against the original to make sure there was no procedural error in the storage.